bin/irc-notify.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+set -eo pipefail
+
+# Required environment variables if using --stage:
+# BRANCH_NAME, BUILD_NUMBER
+
+# defaults and constants
+NICK="irlbot"
+CHANNEL="#www"
+SERVER="irc.mozilla.org:6697"
+BLUE_BUILD_URL="https://ci.us-west.moz.works/blue/organizations/jenkins/irlpodcast"
```

Comment:
I copied this file from https://github.com/mozmar/basket/blob/master/bin/irc-notify.sh and modified the vars above.